identity/app/controllers/editprofile/EditProfileFormHandling.scala

```diff
@@ -14,12 +14,12 @@ trait EditProfileFormHandling extends EditProfileControllerComponents {
   import authenticatedActions._
 
   def displayForm(
-    page: IdentityPage,
-    consentsUpdated: Boolean = false,
-    consentHint: Option[String] = None): Action[AnyContent] = {
+                   page: IdentityPage,
+                   consentsUpdated: Boolean = false,
+                   consentHint: Option[String] = None): Action[AnyContent] = {
```

Comment:
how do i stop intellij from doing this? I _hate_ it and it's not even the guardian's code style

Reply:
https://stackoverflow.com/questions/26880677/intellij-scala-plugins-case-class-indentation-is-absurd